lib/spack/spack/solver/asp.py

```diff
@@ -456,7 +457,16 @@ def __init__(self, cores=True, asp=None):
             asp (file-like): optional stream to write a text-based ASP program
                 for debugging or verification.
         """
-        assert clingo, "PyclingoDriver requires clingo with Python support"
+        global clingo
+        if not clingo:
+            # TODO: Find a way to vendor the concrete spec
+            # in a cross-platform way
+            clingo_spec = spack.spec.Spec('clingo+python')
```

Comment:
```suggestion
            clingo_spec = spack.spec.Spec('clingo@master+python')
```
?

Reply:
I think `@spack` would be better, but yes I should make that change.